src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -127,19 +127,20 @@ public class MempoolValidator : IMempoolValidator
 
         /// <summary>Coin view of the memory pool.</summary>
         private readonly CoinView coinView;
+        private readonly IConsensusRules consensusRules;
```

Comment:
Space and a comment to keep in line with the rest of the class you can use ` /// <inheritdoc seeref="IConsensusRules" />`

Reply:
done